### PR TITLE
Feat: Context tags for subnets and routetables

### PR DESCRIPTION
**Pull request by brad-haynes**
## what
* Context tags added to private and public subnets
* Context tags added to private and public route tables

## why
* Tags were not being passed from context to the subnets and route tables

## references
* No supporting issues/documentation

### CODE REVIEW

File: private.tf [hunk 1]
```diff
@@ -18,7 +18,7 @@ resource "aws_subnet" "private" {
   cidr_block              = cidrsubnet(var.cidr_block, ceil(log(var.max_subnets, 2)), count.index)
   map_public_ip_on_launch = var.map_public_ip_on_launch_enabled
 
-  tags = merge({
+  tags = merge(module.private_label.tags, {
     "Name"      = "${module.private_label.id}${module.this.delimiter}${element(var.subnet_names, count.index)}"
     "Stage"     = module.private_label.stage
```
Comment (Gowiem) [inline private.tf:23]: Can you remove the Stage + Namespace entries from the override map? They're unnecessary now that we're using the top level tags as the base map. Please remove them from all of the instances that you've updated.

**Review by Gowiem — CHANGES_REQUESTED** [review]
@brad-haynes this is good stuff -- Thanks for adding it. One request, but otherwise we'll get this merged.

File: private.tf (hunk 1 above)
Comment (brad-haynes) [reply private.tf]: I was actually just doing that.

**Review by Gowiem — APPROVED** [review]
Good stuff @brad-haynes

**Comment by Gowiem** [comment]
@brad-haynes Thanks for the contribution! Released as https://github.com/cloudposse/terraform-aws-named-subnets/releases/tag/0.12.0

**Comment by brad-haynes** [comment]
Thanks, @Gowiem!

**Pull request merged**